### DISCUSSION TITLE
Warning: Did not expect server HTML to contain a

### DISCUSSION THREAD

**Discussion by dinhtheloc** [vercel/next.js#17412] [Help]
Hello everyone, i have problem when rendering from server side. 
<img width="840" alt="Screen Shot 2020-09-28 at 16 16 09" src="https://user-images.githubusercontent.com/25575696/94413948-f402fc00-01a5-11eb-9053-873457a439b2.png">
I realized that the problem is due to the difference between ssr and client when client side run script jarallax
<img width="1680" alt="Screen Shot 2020-09-28 at 16 20 09" src="https://user-images.githubusercontent.com/25575696/94414468-94f1b700-01a6-11eb-810c-9a4600d3f499.png">
What do I need to do to resolve this error?

**Comment by 3rj** [comment]
Same problem here!
Did you find any solution?
